build.gradle

```diff
@@ -80,8 +80,8 @@ subprojects { project ->
   sourceCompatibility = JavaVersion.VERSION_1_8
   targetCompatibility = JavaVersion.VERSION_1_8
 
-  
-  apply plugin: 'com.jaredsburrows.checkerframework'
+  apply plugin: 'org.checkerframework'
+  //apply plugin: 'com.jaredsburrows.checkerframework'
```

Comment:
Probably should just delete this line rather than commenting it out, since we have no intention of going back to that plugin